modules/test/playwright/tests/login-web/createAccount.spec.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {captchaConfigPageTest} from '../../fixtures/captchaConfigPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import {performLogout} from '../../utils/performLogin';
+
+export const test = mergeTests(loginTest(), captchaConfigPageTest);
+
+test.describe('Duplicate account creation', () => {
+	test('Using already existing email address does not throw failure alert', async ({
+		captchaConfigPage,
+		page,
+	}) => {
+		await captchaConfigPage.goTo();
+
+		await captchaConfigPage.disableCreateAccountCaptcha();
+
+		await captchaConfigPage.saveConfiguration();
```

Comment:
We are already saving the configuration in the `captchaConfigPage.disableCreateAccountCaptcha()` call, so you should remove this line.